utils/mock-data-loader/gef/eligibilityCriteriaAmendments.js

```diff
@@ -0,0 +1,43 @@
+const { FACILITY_TYPE } = require('@ukef/dtfs2-common');
+
+const ELIGIBILITY_CRITERIA_AMENDMENTS = [
+  {
+    version: 1,
+    product: 'GEF',
+    isInDraft: false,
+    facilityType: [FACILITY_TYPE.CASH, FACILITY_TYPE.CONTINGENT],
+    createdAt: 1649876028968,
+    criteria: [
+      {
+        id: 1,
+        text: 'The Facility is not an Affected Facility',
+      },
+      {
+        id: 2,
+        text: 'Neither the Exporter, not its UK Parent Obligor is an Affected Person',
+      },
+      {
+        id: 3,
+        text: 'The Cover Period of the Facility is within the Facility Maximum Cover Period',
+      },
+      {
+        id: 4,
+        text: 'The Covered Facility Limit (converted for this purpose into the Master Guarantee Base Currency) of the Facility is not more than the lesser of (i) the Available Master Guarantee Limit; and the Available Obligor(s) Limit',
```

Comment:
Should this be sub-bullet point?

Reply:
No, in the designs it isn't: 

![image](https://github.com/user-attachments/assets/4dd34e5a-56fe-4a40-937f-efde7fa5e202)